packages/module/patternfly-docs/content/extensions/chatbot/examples/demos/Chatbot.md

```diff
@@ -103,3 +103,11 @@ This demo displays an embedded ChatBot. Embedded ChatBots are meant to be placed
 ```js file="./EmbeddedChatbot.tsx" isFullscreen
 
 ```
+
+### Comparing ChatBots
+
+You may want to compare output from different chatbots based on the same prompt. The following demo is a [pattern](https://www.patternfly.org/patterns/about-patterns/) that could be followed to build a comparison view using the PatternFly ChatBot extension.
```

Comment:
```suggestion
To let users compare how different ChatBots respond to the same prompt, you can add multiple ChatBots within the same window. The following demo illustrates a comparison view pattern that allows users to toggle between different conversations in a single ChatBot window.
```
Just a little rephrasing. If we start to build out chatbot patterns as much as our main pf patterns, then maybe we could think about doing something like adding a separate patterns section within the chatbot docs 🤔 longer term